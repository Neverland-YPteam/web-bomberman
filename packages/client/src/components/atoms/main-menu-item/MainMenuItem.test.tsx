import React from 'react'
import { create } from 'react-test-renderer'
import { MainMenuItem } from './index'
import { BrowserRouter } from 'react-router-dom'

const item = {
  title: 'Title',
  path: 'title'
}

it('MainMenuItem snapshot', () => {
  const actualSnapshot = create(
    <BrowserRouter>
      <MainMenuItem item={item} />
    </BrowserRouter>).toJSON()

  expect(actualSnapshot).toMatchSnapshot()
})
