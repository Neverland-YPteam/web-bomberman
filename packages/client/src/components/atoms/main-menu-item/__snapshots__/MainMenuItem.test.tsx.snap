// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenuItem snapshot 1`] = `
<a
  className="MuiBox-root css-14hnrkt"
  href="/title"
  onClick={[Function]}
>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
  >
    Title
  </h6>
</a>
`;
